snippets/logging6x-loki-statement-snip.adoc

```diff
@@ -0,0 +1,9 @@
+// Text snippet included in the following assemblies:
+// * observability/logging/log_storage/about-log-storage.adoc
+//
+// Text snippet included in the following modules:
+//
+//
+:_mod-docs-content-type: SNIPPET
+
+Loki is a horizontally scalable, highly available, multi-tenant log aggregation system offered as a GA log store for {logging} {for} that can be visualized with the OpenShift {ObservabilityShortName} UI. The Loki configuration provided by OpenShift {logging-uc} is a short-term log store designed to enable users to perform fast troubleshooting with the collected logs. For that purpose, the {logging} {for} configuration of Loki has short-term storage, and is optimized for very recent queries. For long-term storage or queries over a long time period, users should look to log stores external to their cluster.
```

Comment:
Please ignore this file (Not sure how it got added in this commit). I will remove this file in next commit.